phpstan-baseline.neon

```diff
@@ -17233,3 +17228,31 @@ parameters:
 			message: "#^Method Drupal\\\\social_flexible_group_book\\\\Plugin\\\\ActivityEntityCondition\\\\BookParentOnlyActivityEntityCondition\\:\\:isValidEntityCondition\\(\\) has parameter \\$entity with no type specified\\.$#"
 			count: 1
 			path: modules/social_features/social_group/modules/social_group_flexible_group/modules/social_flexible_group_book/src/Plugin/ActivityEntityCondition/BookParentOnlyActivityEntityCondition.php
+
+		-
+			message: "#^The \"social_event\\.enroll\" service is deprecated in social\\:11\\.4\\.0 and is removed from social\\:12\\.0\\.0\\. See https\\://github\\.com/goalgorilla/open_social/pull/3084$#"
```

Comment:
@nechai Please check PHPStan. Looks like we need to update more rules.

Reply:
Added